packages/react-dom/src/__tests__/ReactDOMFizzServer-test.js

```diff
@@ -2841,4 +2841,247 @@ describe('ReactDOMFizzServer', () => {
       expect(window.__test_outlet).toBe(1);
     });
   });
+
+  // @gate experimental && enableClientRenderFallbackOnTextMismatch
+  it('#24384: Suspending should halt hydration warnings while still allowing siblings to warm up', async () => {
```

Comment:
Can we add a test, where there is no mismatch at all between client and server?

Reply:
yes, thanks: https://github.com/facebook/react/pull/24436